jenkinsapi/jenkins.py

```diff
@@ -222,6 +223,10 @@ def delete_view_by_url(self, str_url):
         self.poll()
         return self
 
+    def get_label_by_url(self, label_url):
```

Comment:
Yeah, I noticed that I could just use the baseurl, have updated get_label_by_url to get_label function. :-)